### DISCUSSION TITLE
Cant type password for trades

### DISCUSSION THREAD

**Discussion by asfghnijouw** [dyc3/steamguard-cli#217] [Q&A] [answered]
I cant type password in cmd, what should i do?

**Comment by dyc3** [comment, marked as answer]
The password prompt intentionally doesn't show your password as you type it. It's still there, just type it or paste it as normal and press enter.

Also note that `Ctrl + V` does not paste in most terminals

**Reply by asfghnijouw** [reply]
![image](https://github.com/dyc3/steamguard-cli/assets/106036444/7e13e8fd-3432-4e28-b269-9ab5deb59007)

**Reply by dyc3** [reply]
This looks like a different issue. Can you try again with debug logging enabled? Add `-v debug` to the command.

**Comment by asfghnijouw** [comment]
didnt work
